css/components/button.css

```diff
@@ -13,13 +36,101 @@
 }
 
 .btn-primary:hover{
-    background-color: var(--light-blue);
+    background-color: var(--light-more-blue);
+    box-shadow: 1px 1px var(--light-black);
+}
+
+.btn-success{
+    background-color: var(--success-color);
+    color: var(--mint-white);
+}
+
+.btn-success:hover{
+    background-color: hsl(144,100%,50%);
+    box-shadow: 1px 1px var(--light-black);
+}
+
+.btn-warning{
+    background-color: var(--warning-color);
+    color: var(--mint-white);
+}
+
+.btn-warning:hover{
+    background-color: hsl(40,100%,65%);
+    box-shadow: 1px 1px var(--light-black);
```

Comment:
box-shadow property may be a reduandacy. you have defined it on .btn class and here.

Reply:
i change that you told me